ale/drivers/mex_drivers.py

```diff
@@ -422,3 +495,85 @@ def sensor_model_version(self):
           ISIS sensor model version
         """
         return 1
+
+class MexHrscIsisLabelNaifSpiceDriver(LineScanner, IsisLabel, NaifSpice, RadialDistortion, Driver):
+
+        @property
+        def instrument_id(self):
+            if(super().instrument_id != "HRSC"):
+                raise Exception ("Instrument ID is wrong.")
```

Comment:
I know this isn't necessarily part of this PR but if we could add this to the Pds3LabelNaifSpice driver for Mex it will save us some headache down the road. The property is on between line 159 and 170 in the mex_drivers.py file.